packages/components/stories/List.js

```diff
@@ -375,6 +375,24 @@ storiesOf('List', module)
 			<List {...props} displayMode="large" virtualized />
 		</div>
 	))
+	.add('Virtualized - large with onClick', () => {
```

Comment:
It make sense to separate the displays in VirtualizedList because they are developped in its components. But here, for List component, I would prefer to add all display in the same story that covers the same feature like in the `empty list` story.
* Remove this one
* In activation/rowClick story, add `<List {...selectedItemsProps} displayMode="large" virtualized />` to display both displayMode